### DISCUSSION TITLE
Style not working in plugin development

### DISCUSSION THREAD

**Discussion by kagiyanagi** [gxanshu/grapesjs-user-blocks#2] [Q&A] [answered]
i'm going to fork this repo and suggest some changes so that this plugin have more cool features. but as i read your code everything is well documented but in development mode there is no way you can testing your development CSS.

how can i see changes that i made in CSS file?

**Comment by gxanshu** [comment, marked as answer]
This plugin is ready for production use but still i'm going to make some changes for better developer experience. for now you can see your changes in your browsers by just copy the CSS code and paste it into any CSS file in source of browser page.

here is an article that help you [how-to-edit-css-using-google-chrome-developer-tools](https://www.inmotionhosting.com/support/website/google-tools/how-to-edit-css-using-google-chrome-developer-tools/)

**Reply by kagiyanagi** [reply]
Thanks
